unattended_installer/wazuh_install.sh

```diff
@@ -358,6 +358,11 @@ function main() {
     fi
     checkArguments
 
+    if [ -n "${AIO}" ] || [ -n "${indexer}" ] || [ -n "${dashboard}" ] || [ -n "${wazuh}" ]; then
```

Comment:
Better "if not certificates or start cluster", isn't it?

Reply:
Yes, this simplifies the interpretation. There I add this modification.